docs/building-a-form/available-components.md

```diff
@@ -204,15 +265,47 @@ This indicates to the user that they have either not filled out a required field
 
 #### Usage guidelines
 
-- ?
+There are several ways in which form fields can be invalid: a required field in blank, the entry is too short or long, the entry does not satisfy a specific format, etc.
 
-### Password
+To show an error on a blank field that is required, include the field in the array under the `required` property in the `schema`. An error on that field will automatically be rendered if the field is blank.
```

Comment:
To tie these two points (lines 270 and 272) more closely to the expository paragraph in 268, can you make them bullets under that paragraph, like so?

> There are several ways in which form fields can be invalid, such as a blank required field, the entry is too short or long, or the entry does not satisfy a specific format.
> - **To show an error on a blank field that is required,** include the field in the array under the `required` property in the `schema`. An error on that field will automatically be rendered if the field is blank.
> - **To show an error on a field for any other reason** (e.g., it has not met certain data requirements), pass a validation function to the array for the `ui:validations` property under that field in `uiSchema`.

☝️ I added the bold there to further differentiate between the two options; let me know if that seems like too much.